src/snowflake/cli/api/commands/project_initialisation.py

```diff
@@ -57,6 +58,8 @@ def init(
     init.__doc__ = (
         f"Initializes this directory with a sample set "
         f"of files for creating a {project_type_doc} project."
+        f"This command is deprecated and will be removed soon."
```

Comment:
Maybe `and will be removed in 3.0.0 version`?

Reply:
Let's not add such detailed info, just in case we decide otherwise